src/reducers/notificationsReducer.ts

```diff
@@ -1,21 +1,69 @@
-export enum ActionTypes {
-  ALERT_SHOW = "ALERT_SHOW",
+export interface Notification {
+  id: number;
+  message: string;
+  timestamp: Date;
 }
 
-export interface INotificationsState {
-  alert: string;
+export type NotificationState = Notification[];
+
+export type NotificationAction = ShowNotification | DismissNotification;
+
+export interface ShowNotification {
+  type: "Notification/Show";
+  message: string;
+  timestamp: Date;
+}
+
+export interface DismissNotification {
+  type: "Notification/Dismiss";
+  id: number;
 }
 
-export const initialState: INotificationsState = {
-  alert: "",
-};
+const example: NotificationState = [
+  {
+    id: 0,
+    message: "Hello 1",
+    timestamp: new Date(new Date().getTime() - 30 * 1000),
+  },
+  {
+    id: 1,
+    message: "Hello 2",
+    timestamp: new Date(new Date().getTime() - 60 * 1000),
+  },
+  {
+    id: 2,
+    message: "Hello 3",
+    timestamp: new Date(),
+  },
+];
+
+export const notificationsReducer =
+  (state: NotificationState = example, action: NotificationAction) =>
+    action.type === "Notification/Show" ?
```

Comment:
you like this better than a switch statement? id like to be consistent in whatever we do

Reply:
Yes :). But I agree, let's be consistent.